.github/workflows/check-commit.yml

```
@@ -0,0 +1,31 @@
+name: Check Commit
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check-signed-off-by:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: npm install
+
+      - run: |
+          ROLL_BACK_COUNT=$(( ${{github.event.pull_request.commits}} - 1 ))
```

```suggestion
          ROLL_BACK_COUNT=$(( ${{ github.event.pull_request.commits }} - 1 ))
```

GitHub Actions typically uses `${{` with space. :)